### ISSUE TITLE
Rails start guide uses rbenv

### ISSUE THREAD

**Issue by strzibny** [developer-portal/content#222]
Just saw this page:
https://developer.fedoraproject.org/start/sw/web-app/rails.html

We should be linking this page:
https://developer.fedoraproject.org/tech/languages/ruby/ruby-installation.html

And not putting there rbenv:(

rbenv, chruby and others can be link for reference. There is no real reason of suggesting a version manager unless user needs it.

**Comment by strzibny** [comment]
@pvalena did I missed something?

**Comment by pvalena** [comment]
You are right. `dnf groupinstall @rubyonrails`, or similar, should be recommended instead.

Feel free to submit PR.

**Comment by strzibny** [comment]
@pvalena I feel like the page is just a duplicate of the Ruby and RoR pages (https://developer.fedoraproject.org/tech/languages/ruby/ror-installation.html) I wrote before and don't want to copy paste it. Perhaps there could be links to:

- install Ruby
- install Ruby on Rails

and then only keeping the first basic commands to set it up. At least if rbenv is packaged or provided as Copr I would be confident enough to suggest it as alternative...

**Comment by strzibny** [comment]
Would like to know some opinion of @voxik

**Comment by pvalena** [comment]
Yes, that all seems the way to go.
I did not mean duplicating the content, but using a reference, like the ones you wrote, instead.

**Comment by yashraj-Dhanuka** [comment]
I am a second year Engineering student . I want to contribute in this Organization.Any help given from mentors will be very helpful to me on how to contribute. i am a novice in this

**Comment by pvalena** [comment]
@yashraj-Dhanuka, please check our [contributing guide](http://developer-portal.github.io/contributing/) and join our IRC channel #developer-portal on [Freenode](https://freenode.net/), as well as our [mailing list](https://lists.fedoraproject.org/admin/lists/developer-portal.lists.fedoraproject.org/).
Feel free to submit PRs for any issue you like and do not hesitate to ask if you encounter some problems.

**Comment by strzibny** [comment]
@yashraj-Dhanuka are you willing to do a PR for this very issue? If yes, you can start and I would be here to provide any help.
